src/dashboard/frontend/transfer-browser/app/front_page/content.html

```diff
@@ -22,22 +22,20 @@
   <div class="help-block">{{ "Accession no." | translate }}</div>
   </div>
 
-  <div class="col-xs-1">
+  <div class="col-xs-3">
   <button type="submit"
           class="btn btn-default"
           data-toggle="collapse"
           data-target="#transfer_browse_tree"
           aria-expanded="false"
-          aria-controls="transfer_browse_tree">
-    {{ "Browse" | translate }}
-  </button>
-  </div>
-
-  <div class="col-xs-2">
```

Comment:
I've put the "Browse" and "Start transfer" buttons together in the same column (x3) so when the "Approve automatically" checkbox overflows there's more real estate for the text to be presented.

To be clear:

![image](https://user-images.githubusercontent.com/606459/43110438-cd098f58-8ea0-11e8-96cd-344c5f643a4e.png)

Instead of:

![image](https://user-images.githubusercontent.com/606459/43110452-e58451d0-8ea0-11e8-9891-274ac7272a1f.png)